src/view/explorer/scene-menu-items.ts

```diff
@@ -5,6 +5,7 @@ import { get } from "svelte/store";
 
 const getSelectedDraftWithIndex = () => {
     const draft = get(selectedDraft) as MultipleSceneDraft;
+    if (draft == null) return { index: -1, draft };
```

Comment:
Don't use double equals, wrap if body in curly brackets

```suggestion
    if (!draft) {
        return { index: -1, draft };
    }
```